x-pack/plugins/observability_solution/observability_onboarding/public/assets/auto_detect.sh

```diff
@@ -333,7 +333,9 @@ detect_known_integrations() {
     fi
   done
 
-  if compgen -G "/var/lib/docker/containers/*/*-json.log" > /dev/null; then
+  if [ -S /var/run/docker.sock ]; then
```

Comment:
This fixes docker detection on macOS where only metrics are available

<img width="1117" alt="Screenshot 2024-07-28 at 19 33 42" src="https://github.com/user-attachments/assets/ab629dcf-1581-4f01-920e-c013f2b7cd80">

Reply:
Nice drive-by fix!